internal/collect/hosts.go

```diff
@@ -74,7 +68,7 @@ func Hosts(config *config.Config) {
 			collectedData := config.PerfCollector.Collect(hostsRefs, metricsToCollect, performance.RealTimeInterval)
 			dc.AddPerfMetrics(collectedData)
 
-			logger.WithField("seconds", time.Since(now).Seconds()).Debug("hosts perf metrics collected")
+			logger.WithField("seconds", config.Uptime()).Debug("hosts perf metrics collected")
```

Comment:
i think Uptime is affected by the time spent at datacenter collection and tags in the collect.go, so we should take that into consideration when looking at this debugging prints

Reply:
Do we want to show the total time since the beginning of the integration, of the time of this particular operation?
This Uptime() is calculated on the start time of the integration, so it will always be increasing. 
I think the "now" that we were using before was coming from "config.Now" so it is the same value, i just put the calculation in the function instead of doing it "manually" everytime.

Reply:
i think either way is ok , but since the datacenter+tags are collected before all the collectors start we just need to take that into account. In this case `now := time.Now()` was at the beginning of the func, so i guess it was using the time since the func started.